environments/extractDiffs.toml

```diff
@@ -215,7 +227,59 @@
         deployed = 10322821
+    [contract.OSM_BAL]
```

Comment:
Don't think we want these OSM contracts in here - not transforming storage from them, and they're fairly noisy since they're regularly providing price updates

Reply:
Happy to take them out, but we should probably find a way to make this more intuitive in the code. There's no real way to tell which contracts go where.

Reply:
Agreed - got any ideas? Best I can come up with is comparing against the storage transformers :/

Reply:
I don't know that I understand how these processes work enough to have clear suggestions. I have questions thuogh.

If I understand this right for execute (and backfills) we use the generated TransformerExporter to determine what to transform - it's a loop. So if a storage contract we don't care about is in the contracts section of the config, it won't matter.  This is why those are empty for the backfills unless we explicitly want to do a backfill.

But extract-diffs will extract diffs based on what's in the contract section. So for execute contracts is just a lookup for info, but for extract-diffs it actually determines what diffs to extract in addition. 

If I have that right we could have an explicit list of what to listen for in the config. It'll have some duplication, but then we could pull the contract section out.

Reply:
I like it! For reference - this is the code that grabs the addresses from this config file for the `extractDiffs` process: https://github.com/makerdao/vulcanizedb/blob/prod/libraries/shared/streamer/filter_query.go#L31